__tests__/__integration__/extension.integration.test.ts

```diff
@@ -603,78 +603,170 @@ describe("Extension Integration Tests", () => {
     describe("Migrating a data set", () => {
         describe("Success Scenarios", () => {
             describe("Migrate a sequential data set", () => {
-                const dataSetName = `${pattern}.SDATA.SET`;
+                const dataSetName = `${pattern}.SDATA.MIGR`;
 
                 beforeEach(async () => {
                     await zowe.Delete.dataSet(sessionNode.getSession(), dataSetName).catch((err) => err);
                     await zowe.Create.dataSet(
-                        sessionNode.getSession(),
-                        zowe.CreateDataSetTypeEnum.DATA_SET_SEQUENTIAL,
-                        dataSetName,
-                    );
+                            sessionNode.getSession(),
+                            zowe.CreateDataSetTypeEnum.DATA_SET_SEQUENTIAL,
+                            dataSetName,
+                        ).catch((err) => err);
```

Comment:
I think this has been indented once too many and I dont think this catch is needed

Reply:
Yep, the catch isn't need there, good spot (again)